.github/workflows/runtimes-matrix.json

```diff
@@ -74,17 +81,19 @@
     "package": "collectives-westend-runtime",
     "path": "cumulus/parachains/runtimes/collectives/collectives-westend",
     "header": "cumulus/file_header.txt",
-    "bench_features": "runtime-benchmarks",
     "template": "cumulus/templates/xcm-bench-template.hbs",
+    "bench_features": "runtime-benchmarks",
+    "bench_flags": "",
     "uri": "wss://westend-collectives-rpc.polkadot.io:443"
   },
   {
     "name": "contracts-rococo",
     "package": "contracts-rococo-runtime",
     "path": "cumulus/parachains/runtimes/contracts/contracts-rococo",
     "header": "cumulus/file_header.txt",
-    "bench_features": "runtime-benchmarks",
     "template": "cumulus/templates/xcm-bench-template.hbs",
+    "bench_features": "runtime-benchmarks",
+    "bench_flags": "--genesis-builder-policy=none --exclude-pallets=pallet_xcm",
```

Comment:
Hmm I have to say I don't really understand why we are using `genesis-builder-policy=none`. It will use an empty genesis, why do we want that?

Reply:
We use none for the runtimes that [dont yet support](https://github.com/paritytech/polkadot-sdk/issues/5704) the genesis builder pattern. It is just to make the benchmarks run at all, otherwise it just errors that there are no presets. That is why i only added it to some runtimes.

Reply:
After DMin i will try it with the chain specs, maybe those work.

Reply:
The specs are not compiled with the benchmarking features, so that does not work either.